homeassistant/components/webostv/helpers.py

```diff
@@ -55,7 +55,8 @@ def async_get_client_by_device_entry(
     Raises ValueError if client is not found.
```

Comment:
What do we use this one for again? Ideally we also check if the entry is loaded, but that can also be a followup

Reply:
Will do a follow-up to check if entry is loaded